build-system/tasks/bundle-size.js

```diff
@@ -36,11 +35,34 @@ const expectedGitHubProject = 'ampproject/amphtml';
 
 const {green, red, cyan, yellow} = colors;
 
+/**
+ * Get the max bundle size from the build artifacts repository.
+ *
+ * @return {number} the max allowed bundle size.
+ */
+async function getMaxBundleSize() {
+  return await octokit.repos.getContent(
+      Object.assign(buildArtifactsRepoOptions, {
+        path: path.join('bundle-size', '.max_size'),
+      })
+  ).then(result => {
+    const maxSize =
+        Buffer.from(result.data.content, 'base64').toString().trim();
+    log('Max bundle size from GitHub is', cyan(maxSize));
+    return maxSize;
```

Comment:
`+ MAX_INCREMENTAL_SIZE`;

Reply:
I want to do that part in a followup CL :)

Reply:
Let's do it now to avoid making the bundle-size check more strict (read: painful) from this PR.

Also we need a bypass for PRs that intentionally increase the size substantially.

Reply:
Agree with @choumx, since this has been so painful for folks it makes sense to move a little slower with more completeness.

Reply:
+1 to adding an allowance for an increase in size, so PRs based on this change aren't blocked.

Reply:
Discussed this offline with @choumx. Basically, this PR is not going to change the current logic that determines whether the bundle size is going above cap, it just changes where the "hard coded" value is coming from (that is, moving it from the `bundle-size.js` file into the artifacts repository) - the allowance is indeed part of the next step for this, but Will and I agree that smallers PRs are better, so I'm going to merge this and immediately send the next PR :)